controllers/post-comments-controller.js

```diff
@@ -0,0 +1,14 @@
+//REQUIRE
+const {newComment} = require("../models/post-comment-model")
+//FUNCTION
+function postComments(req, res, next) {
+  newComment(req.params.article_id, req.body)
+      .then((addedComment) => {
+      return res.status(201).send(addedComment);
```

Comment:
Don't forget we want to send the data back on a relevant key name, for example: `.send({ comment: addedComment });`
_note: this will change how you access the data in the response for your tests_